File: methodOR.java
class mainclass{
    int a=10;
    void displaymsg(){
        System.out.println(++a);
    }
}
class subclass extends mainclass{
    void displaymsg(){
        System.out.println(a);
        super.displaymsg();
    }
}

class methodOR{
    public static void main(String[] args){
        subclass obj = new subclass();
        obj.displaymsg();
    }
}

File: methodOR.cpp
#include<iostream>

using namespace std;

class mainclass{
    public:
    int a=10;
    void displaymsg(){
        cout<<++a<<endl;
    }
};
class subclass : public mainclass{
    public:
    void displaymsg(){
        cout<<a<<endl;
    }
};

int main(){
    subclass obj;
    obj.displaymsg();
}
